app/build.gradle

```diff
@@ -143,6 +143,7 @@ android {
             resValue "string", "app_name", "Tree Tracker(dev)"
             buildConfigField "Boolean", "GPS_ACCURACY", "false"
             buildConfigField 'String', 'BASE_URL', '"http://dev.treetracker.org/api/mobile/"'
+            buildConfigField 'String', 'PLANTER_ACCOUNT_URL', '"http://192.168.2.128:3000/"'
```

Comment:
Please note, this is a dev url used to test the changes with a locally running planter account api. This needs to be updated when a dev version of the account payment service is available